Proposal to add a README file. (#10)
jenkinsci/ui-samples-plugin: README.md

# Jenkins UI sample plugin

> This plugin helps Jenkins developpers to build user interface using Jenkins UI components.

Reviewer: ```suggestion
This fake plugin shows Jenkins developers how to build user interfaces using Jenkins UI components.
```

This plugin contains:
- demonstration of UI controls available in Jenkins based on Stapler, Jelly, Groovy, etc.

> - snippets of jelly/groovy "ready-to-use" examples

Reviewer: ```suggestion
- snippets of Jelly/Groovy "ready-to-use" examples
```

## Usage

Run `mvn hpi:run`

> Go on "UI Samples" menu item or at http://localhost:8080/jenkins/ui-samples/ and play with UI components :).

Reviewer: ```suggestion
Go to the **UI Samples** menu item or straight to http://localhost:8080/jenkins/ui-samples/ and play with UI components.
```
